pkg/controller/garbagecollector/metrics.go

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Yet another client... Can you revisit it here? Those metrics are important when we encounter performance regression. We saw a number of them caused by client rate limiter saturation.